src/utils/quickPickWorkspaceFolder.ts

```diff
@@ -16,6 +17,15 @@ export async function quickPickWorkspaceFolder(noWorkspacesMessage: string): Pro
         }
         return selected;
     } else {
+        context.errorHandling.suppressReportIssue = true;
+        context.errorHandling.buttons = [
+            {
+                callback: async () => {
```

Comment:
Maybe. Probably. ☺️ I assume we would also need to have means to pick a folder in case their workspace is multi-root?

Reply:
Yeah, the way I see it either it should have "Open Folder" or "Open Workspace" + "Open Folder". I'd prefer users stick to single root, i.e. "Open Folder", because everything works better in that situation, which is why I'm a little reluctant to have "Open Workspace".

Reply:
Makes sense